### PR TITLE
WS-1516 added python documentation

### PR DESCRIPTION
**Pull request by markussWs**

### CODE REVIEW

File: docs/API/download-scraped-data-in-csv-format.code.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+output_file = f'/tmp/scrapingjob-data{scraping_job_id}.csv'
+
+client.download_scraping_job_csv(scraping_job_id, output_file)
+
+# Use a library that supports RFC 4180 standard to parse the csv file.
+# That said we recommend downloading data and using JSON format data.
+# CSV format readers and writers have been incorrectly implemented in multiple applications and programming languages.
+
+# remove temporary file
+os.remove(output_file)
```
Comment (kristapsws) [inline docs/API/download-scraped-data-in-csv-format.code.py:13]: add os import at the start of the sample

File: docs/API/download-scraped-data-in-json-format.code.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+client.download_scraping_job_json(scraping_job_id, output_file)
+
+# read data from file with built in JSON reader
+reader = JsonReader(output_file)
+
+for record in reader.fetch_rows():
+    print(f"ROW: {record} \n")
+
+# remove temporary file
+os.remove(output_file)
```
Comment (kristapsws) [inline docs/API/download-scraped-data-in-json-format.code.py:15]: same as for csv

**Review by kristapsws — CHANGES_REQUESTED** [review]

File: docs/API/download-scraped-data-in-csv-format.code.py (hunk 1 above)
Comment (markussWs) [reply docs/API/download-scraped-data-in-csv-format.code.py:13]: done

File: docs/API/download-scraped-data-in-json-format.code.py (hunk 2 above)
Comment (markussWs) [reply docs/API/download-scraped-data-in-json-format.code.py:15]: done

**Review by kristapsws — APPROVED** [review]

**Comment by martinsbalodis** [comment]
We will use OpenAPI instead of this.

**Pull request closed without merging**